src/porepy/fracs/fractures_2d.py

```diff
@@ -150,17 +150,35 @@ def add_network(self, fs):
         return FractureNetwork2d(p, e, domain, self.tol)
 
     def mesh(self, mesh_args, tol=None, do_snap=True, constraints=None, **kwargs):
+        """ Create GridBucket (mixed-dimensional grid) for this fracture network.
+
+        Parameters:
+            mesh_args: Arguments passed on to mesh size control
+            tol (double, optional): Tolerance used for geometric computations.
+                Defaults to the tolerance of this network.
+            do_snap (boolean, optional): Whether to snap lines to avoid small
+                segments. Defults to True.
+            constraints (np.array of int): Index of network edges that should not
+                generate lower-dimensional meshes, but only act as constraints in
+                the meshing algorithm.
+
+        Returns:
+            GridBucket: Mixed-dimensional mesh.
+
+        """
 
         if tol is None:
             tol = self.tol
+        if constraints is None:
+            constraints = np.empty(0, dtype=np.int)
 
         p = self.pts
         e = self.edges
 
         if do_snap and p is not None and p.size > 0:
             p, _ = pp.frac_utils.snap_fracture_set_2d(p, e, snap_tol=tol)
-        grid_list = pp.fracs.simplex.triangle_grid(
-            p, e[:2], self.domain, tol=tol, subdomains=constraints, **mesh_args
+        grid_list = porepy.fracs.simplex.triangle_grid(
```

Comment:
Is there a reason we don't import porepy as pp?

Reply:
Me struggling with imports.. I'll make a second attempt